packages/typespec-ts/test/unit/modelsGenerator.spec.ts

```diff
@@ -611,6 +611,7 @@ describe("Input/output model type", () => {
       export interface SimpleModel { 
         "prop1": string;
         "prop2": string;
+        "x": string;
```

Comment:
why the quote here is not removed? I think it should be removed if no special character in between?

Reply:
I reduce quote here, actually both are accepted because we would compare after prettier so they are the same syntax.